tests/string.cpp

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2012-present Facebook, Inc.
+ * Licensed under the Apache License, Version 2.0 */
+
+#include "watchman.h"
+#include <algorithm>
+#include "thirdparty/tap.h"
+
+void w_log(int level, WATCHMAN_FMT_STRING(const char *fmt), ...) {
```

Comment:
please remove this function; there are stubs for this in `log_stub.cpp` that you should use instead